ipaserver/install/dsinstance.py

```diff
@@ -566,17 +565,16 @@ def __create_instance(self):
         inst.local_simple_allocate(
             serverid=self.serverid,
             ldapuri=ipaldap.get_ldap_uri(realm=self.realm, protocol='ldapi'),
-            password=self.dm_password
         )
 
         # local_simple_allocate() configures LDAPI but doesn't set up the
         # DirSrv object to use LDAPI. Modify the DirSrv() object to use
-        # LDAPI with password bind. autobind is not available, yet.
+        # LDAPI with with autobind.
         inst.ldapi_enabled = 'on'
+        inst.ldapi_autobind = 'on'
```

Comment:
It seems a strange API but after setting the ldapi URL you can call init.setup_ldapi() to set ldapi_enabled and ldapi_autobind to "on". One is not better than the other, I just wonder if they'll refactor this at some point.

Reply:
Ah, the function even works without setting the LDAPI url manually.